runtime/parachains/src/paras_inherent.rs

```diff
@@ -107,9 +97,18 @@ fn minimal_inherent_weight<T: Config>() -> Weight {
 		.min(<T as Config>::WeightInfo::enter_backed_candidates_variable(0))
 }
 
-/// Extract the weight that is adder _per_ backed candidate.
-fn backed_candidate_weight<T: Config>(validity_votes_len: u32) -> Weight {
-	<T as Config>::WeightInfo::enter_backed_candidates_variable(validity_votes_len as u32)
+fn dispute_statements_weight<T: Config>(disputes: &[DisputeStatementSet]) -> Weight {
+    disputes.iter().map(|d| <<T as Config>::WeightInfo as WeightInfo>::enter_variable_disputes(d.statements.len() as u32)).sum()
```

Comment:
We need to use `.saturating_add` and sum uses normal additions (see https://doc.rust-lang.org/src/core/iter/traits/accum.rs.html#45), so I would reccomend using a `.fold` with saturating add to sum these